src/CC-BY-4.0.xml

```diff
@@ -5,10 +5,10 @@
       <crossRefs>
          <crossRef>http://creativecommons.org/licenses/by/4.0/legalcode</crossRef>
       </crossRefs>
-      <titleText>
-         <p>Creative Commons Attribution 4.0 International</p>
-      </titleText>
       <optional>
+         <titleText>
+            <p><optional>Creative Commons </optional>Attribution 4.0 International</p>
+         </titleText>
```

Comment:
I moved this inside the optional block because it doesn't make sense to have

```
title
optional
title
body
```

meaning that

```
title
title
body
```

is a reasonable rendering.

And note that "Creative Commons " is not included in the version provided by CC, thus made optional.

Reply:
I'm actually treating all titleText as optional when creating the license matching templates - so we don't really need to wrap the titleText with optional.